mycli/main.py

```diff
@@ -111,6 +114,14 @@ def __init__(self, sqlexecute=None, prompt=None,
         # Register custom special commands.
         self.register_special_commands()
 
+        # Load .mylogin.cnf if it exists.
+        mylogin_cnf_path = get_mylogin_cnf_path()
+        if mylogin_cnf_path:
+            mylogin_cnf = get_mylogin_cnf_plaintext(mylogin_cnf_path)
```

Comment:
Okay, should mycli attempt to continue after printing an error message?

`mysql` isn't much help here because it doesn't even throw an error message! It just silently continues, even if it can't read `.mylogin.cnf`.